docs/environment_creation_from_scratch.md

```diff
@@ -100,13 +107,13 @@ Once setup, run the deployment pipeline in order to update the ECS task definiti
 ### Set up the database
 
 If you're not restoring the database from a backup / migrating it from elsewhere, and the db migration stage failed
-above, run a `db:setup` rake task (using the `ad_hoc` task definition to spin it up from the app image, with a command 
-override).
+in the deployment above, run a `db:setup` rake task (using the `ad_hoc` task definition to spin it up from the app image, 
+with a command override).
```

Comment:
Is your editor doing automatic line breaks or something? (It's not a problem when considering what it looks like as markdown, but it's adding extra diffs)